core/src/main/resources/defaults/blueprints/cdp-data-engineering-hueha-702.bp

```diff
@@ -48,6 +48,10 @@
           {
             "name": "yarn_admin_acl",
             "value": "yarn,hive,hdfs"
+          },
+          {
+            "name": "yarn_service_mapred_safety_valve",
+            "value": "<property><name>mapreduce.shuffle.connection-keep-alive.enable</name><value>true</value></property>"
```

Comment:
Should really really avoid safety valves. If this config is important, why isn't it exposed in CM as a real config value?